packages/frontend/src/i18n/resources/en.json

```diff
@@ -24,123 +24,39 @@
   "dialog.transmission.collapseLabel": "Hide history",
   "search.autocomplete.searchForSender.withQuery": "Search for dialogs from ∫{sender} with query {query}",
   "search.autocomplete.searchForSender.withoutQuery": "Search for dialogs from {sender}",
-  "dialogs.attachment_count": "{count, plural, one {# attachment} other {# attachments}}",
-  "editSavedSearch.give_search_name": "Name the search",
-  "editSavedSearch.save_and_close": "Save and close",
-  "editSavedSearch.search_without_name": "Search without name",
   "error.dialog.not_found": "Dialog not found",
-  "filter.date.last_six_months": "Last six months",
```

Comment:
filter.date.* and filter.query.* are in use, but the * is derived dynamically. Take a look at `getFilterLabel` in `useFilters`